From 1e8efc07613f5fbf59ad7e580132c473a9b1d3eb Mon Sep 17 00:00:00 2001
From: Luka Perkov <luka@openwrt.org>
Date: Sun, 11 Aug 2013 13:13:16 +0200
Subject: [PATCH 2/3] ARM: dts: imx6sl/imx6qdl: use the same SDMA firmware

Patch for SDMA firmware sent by Fabio Estevam in January [1] didn't make
upstream. Nonetheless, it was pointed out that all imx6 use the same SDMA
firmware [2]. Reflect this fact in appropriate .dtsi files.

[1] http://lists.infradead.org/pipermail/linux-arm-kernel/2013-January/143025.html
[2] http://lists.infradead.org/pipermail/linux-arm-kernel/2013-January/143150.html

Signed-off-by: Luka Perkov <luka@openwrt.org>
---
 arch/arm/boot/dts/imx6qdl.dtsi | 2 +-
 arch/arm/boot/dts/imx6sl.dtsi  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -580,7 +580,7 @@
 				interrupts = <0 2 0x04>;
 				clocks = <&clks 155>, <&clks 155>;
 				clock-names = "ipg", "ahb";
-				fsl,sdma-ram-script-name = "imx/sdma/sdma-imx6q.bin";
+				fsl,sdma-ram-script-name = "imx/sdma/sdma-imx6.bin";
 			};
 		};
 
